### PR TITLE
Fix get_tensor_value() issues

### PR DESCRIPTION
**Pull request by pengwa**
fix bug https://github.com/onnx/tensorflow-onnx/issues/263

### CODE REVIEW

**Comment by pengwa** [comment]
@zhijxu-MS

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -1354,11 +1354,11 @@ def onehot_op(ctx, node, name, args):
...
     # axis becomes axis for gather
     node.set_attr("axis", 0)
-    depth = node.inputs[1].get_tensor_value()[0]
-    on = node.inputs[2].get_tensor_value()[0]
-    off = node.inputs[3].get_tensor_value()[0]
-    dtype = node.inputs[2].get_tensor_type()
-    eye = np.eye(depth, dtype=dtype)
+    depth = node.inputs[1].get_tensor_value()
+    on_val = node.inputs[2].get_tensor_value(as_list=False)
+    on = on_val.tolist()
```
Comment (guschmue) [inline tf2onnx/tfonnx.py]: on_val.tolist()  ?

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -1354,11 +1354,11 @@ def onehot_op(ctx, node, name, args):
...
+    off = node.inputs[3].get_tensor_value()
```
Comment (guschmue) [inline tf2onnx/tfonnx.py]: off is a list ?

File: tf2onnx/rewriter/lstm_rewriter.py [hunk 3]
```diff
@@ -73,7 +73,7 @@ def get_weight_and_bias(self, match):
         if not ft:
             return None
 
-        if not (len(ft.value) == 1 and self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0])):
+        if not (ft.value.tolist() == 1 and self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0])):
```
[the remark this answers is not in the document]
Comment (pengwa) [reply tf2onnx/rewriter/lstm_rewriter.py]: Thanks. Just double checked np.array(5) == 5 is also allowed. tolist() is not necessary.

File: tf2onnx/tfonnx.py (hunk 2 above)
Comment (pengwa) [reply tf2onnx/tfonnx.py]: for a scalar, tolist() won't be a real list, it is still a single value. while tolist() changed the return value's type.

File: tf2onnx/tfonnx.py [hunk 4]
```diff
@@ -253,7 +253,7 @@ def arg_minmax_op(ctx, node, name, args):
     # output_type output = ArgMin(T input, Tidx dimension, @type Tidx, @type output_type)
     # tensor(int32) reduced = ArgMin(T data, @INT axis, @INT keepdims)
     axis_node = node.inputs[1]
-    axis = axis_node.get_tensor_value()[0]
+    axis = axis_node.get_tensor_value()
```
[the remark this answers is not in the document]
Comment (guschmue) [reply tf2onnx/tfonnx.py]: I see. A little hard to understand.

[the remark this answers is not in the document]
Comment (pengwa) [reply tf2onnx/tfonnx.py]: yeah, it is. I am adding some comments about the scalar case.

**Pull request merged**